utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/KafkaUniversalComponentTransformer.scala

```diff
@@ -62,7 +73,20 @@ abstract class KafkaUniversalComponentTransformer[T, TN <: TopicName: TopicValid
   protected def getTopicParam(
       implicit nodeId: NodeId
   ): WithError[ParameterCreatorWithNoDependency with ParameterExtractor[String]] = {
-    val topics = topicSelectionStrategy.getTopics(schemaRegistryClient)
+    val fetchAllTopics: Option[Boolean] =
+      if (modelDependencies.config.hasPath("fetchKafkaTopicsWithoutSchema")) {
+        Some(modelDependencies.config.getBoolean("fetchKafkaTopicsWithoutSchema"))
+      } else
+        None
+
+    val topics = getAllTopics(fetchAllTopics) match {
```

Comment:
imho:
1. Let's move fetchKafkaTopicsWithoutSchema to KafkaConfig and use typed config param where needed.
2. Let's use the `fetchKafkaTopicsWithoutSchema` config to select desired `TopicSelectionStrategy` once (sth like `AllNonHIddenTopics` and `ExistingSchemaRegistrySubjects`).
3. Delete `filterTopics` method from `TopicSelectionStrategy` but add kafka `Admin` to `getTopics` method

IMO it'll make the code more straightforward - we'll put logic about deciding what topics are available to single place. 

@ForrestFairy @arkadius wdyt?

Reply:
I aggree but also let's' rename this flag to sth like `showTopicsWithoutSchema`. (`kafka` part is not needed because we are already in kafka config, fetching is a technical, implementation detail that the user is not familiar with)